modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceTest.java

```diff
@@ -562,6 +562,49 @@ public void testAddAndDeleteCompanyWithUserGroupAndUserGroupRole()
 		Assert.assertNull(_userLocalService.fetchUser(user.getUserId()));
 	}
 
+	@Test
+	public void testCheckCompanyDefaultUserLocaleAndTimeZoneDifferFromCompany()
+		throws Exception {
+
+		Company company = addCompany();
+
+		try {
+			User defaultUser = company.getDefaultUser();
+
+			String expectedLanguageId = defaultUser.getLanguageId();
+			String expectedTimeZoneId = defaultUser.getTimeZoneId();
+
+			String langaugeId = "en_US";
+
+			if (langaugeId.equals(expectedLanguageId)) {
+				langaugeId = "fr_FR";
+			}
+
+			String timeZoneId = "UTC";
```

Comment:
@yuhai It seems UTC is not a timezone, but a time standard. Are you sure you should use this?